docs/contributors/releasing.md

```diff
@@ -31,16 +31,20 @@ title: Making a release
 
 First of all, if you are bumping the minor part of the version choose a new
 metal or an alloy as a new for the release! Use that in the release notes and in
-the release title on GitHub. Otherwie, use the metals from the previous release.
+the release title on GitHub. Otherwise, use the metal from the previous release.
 
-You might use the `./bin/merged_prs.sc` script to generate merged PRs list
-between two last release tags. It can be run using scala-cli:
+You might use the `./bin/merged_prs.sc` script to generate a draft of release
+notes with merged PRs list between two last release tags. It can be run using
+scala-cli:
 
 ```
 cs install scala-cli
 scala-cli ./bin/merged_prs.scala -- <tag1> <tag2> "<github_api_token>"
 ```
 
+Make sure that required tags, the previous one and the new one, are available.
+You might need to do `git fetch <main_fork> --tags` to fetch them.
```

Comment:
I feel like that's a bit misleading, "to fetch them" suggests you can fetch both needed tags, where in truth you have to fetch one and create the other one.

Reply:
Yes, thanks 👍